packages/dds/tree/src/test/rebase/generateFuzzyCombinedChange.spec.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { ChangeRebaser } from "../../rebase";
+import { AnchorSet } from "../../tree";
+import { generateFuzzyCombinedChange } from "./fuzz";
+
+const testSeed = 432167897;
+
+type TChange = TChange[] | { I: TChange; } | { C: TChange; O: TChange; } | string;
+
+const testStringRebaser: ChangeRebaser<TChange> = {
+    compose: (changes: TChange[]) => changes,
+    invert: (change: TChange) => ({ I: change }),
+    rebase: (change: TChange, over: TChange) => ({ C: change, O: over }),
+    rebaseAnchors: (anchor: AnchorSet, over: TChange) => {},
+};
+
+function generateRandomChange(seed: number) {
+    return String(seed);
```

Comment:
Now that we're not trying to build a string to represent the combinations, we can just use the number as is.
```suggestion
    return seed;
```
You'll need to update the change type to reflect that.